cmd/agent/Dockerfile

```diff
@@ -8,14 +8,19 @@ ARG IMAGE_TAG
 RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list
 RUN apt-get update && apt-get install -t bullseye-backports -qy libsystemd-dev
 
+# Add support for bcc bindings required to compile the eBPF integration
+RUN apt-get update && apt-get install -qy libbpfcc-dev
+
 RUN make clean && make IMAGE_TAG=${IMAGE_TAG} RELEASE_BUILD=${RELEASE_BUILD} BUILD_IN_CONTAINER=false agent
 
 FROM debian:bullseye-slim
 
 # Backports repo required to get a libsystemd version 246 or newer which is required to handle journal +ZSTD compression
+# plus the libbpfcc library for running the eBPF integration.
 RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list
 RUN apt-get update && apt-get install -t bullseye-backports -qy libsystemd-dev && \
   apt-get install -qy tzdata ca-certificates && \
+  if [ `uname -m` = "x86_64" ]; then apt-get install -qy libbpfcc-dev; fi && \
```

Comment:
(nit) You could simplify this to avoid the need for the if / fi: 

```bash
[ `uname -m` = "x86_64" ] && apt-get install -qy libbpfcc-dev
```

Reply:
This seems strange because the && here will execute regardless of what the if condition evaluates to. Was that the intention? If so, you don't really need this && 

Or was the intention to do something like this?
```
if [ `uname -m` = "x86_64" ] && apt-get install -qy libbpfcc-dev; then 
  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
fi
```